docs/agent-config.adoc

```diff
@@ -0,0 +1,44 @@
+= Agent configuration
```

Comment:
Observation: this section needs to duplicate a lot of what's also in the module documentation. I estimate that DRY is not feasible.